cpu/cc26xx_cc13xx/vectors.c

```diff
@@ -27,21 +28,21 @@ void dummy_handler(void) {
     dummy_handler_default();
 }
 
-/* CC26x0 specific interrupt vectors */
+/* CC26xx_CC13xx specific interrupt vectors */
 WEAK_DEFAULT void isr_edge(void);
 WEAK_DEFAULT void isr_i2c(void);
 WEAK_DEFAULT void isr_rfc_cpe1(void);
-WEAK_DEFAULT void isr_aon_rx_tx_cs(void);
+WEAK_DEFAULT void isr_pka(void);
 WEAK_DEFAULT void isr_aon_rtc(void);
-WEAK_DEFAULT void isr_uart(void);
-WEAK_DEFAULT void isr_scse0_aon(void);
+WEAK_DEFAULT void isr_uart0(void);
+WEAK_DEFAULT void isr_aux0_aon(void);
 WEAK_DEFAULT void isr_ssi0(void);
 WEAK_DEFAULT void isr_ssi1(void);
 WEAK_DEFAULT void isr_rfc_cpe0(void);
 WEAK_DEFAULT void isr_rfc_hw(void);
 WEAK_DEFAULT void isr_rfc_cmd_ack(void);
 WEAK_DEFAULT void isr_i2s(void);
-WEAK_DEFAULT void isr_scse1_aon(void);
+WEAK_DEFAULT void isr_aux1_aon(void);
```

Comment:
Are those the same?

Reply:
Couldn't find SCSE in the datasheet. But it's `AUX software event 1`.